push/service_test.go

```diff
@@ -83,7 +83,7 @@ func TestTimestampError(t *testing.T) {
 
 	handler.HandleFunc("/3/device/", func(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusGone)
-		w.Write([]byte(`{"reason":"Unregistered","timestamp":12622780800}`))
+		w.Write([]byte(`{"reason":"Unregistered","timestamp":1262278080000}`))
```

Comment:
shouldn't this have one more zero?

and then line 108/109 wouldn't need to change.